docs/gen-ai/gen-ai-spans.md

```diff
@@ -84,19 +85,24 @@ If none of these options apply, the `gen_ai.system` SHOULD be set to `_OTHER`.
 the canonical name of exception that occurred, or another low-cardinality error identifier.
 Instrumentations SHOULD document the list of errors they report.
 
-**[4] `gen_ai.output.type`:** This attribute SHOULD be used when the client requests output of a specific type. The model may return zero or more outputs of this type.
+**[4] `gen_ai.conversation.id`:** Instrumentations SHOULD populate conversation id when they have it readily available for a given operation, for example:
+-  when client framework being instrumented manages conversation history (see [LlamaIndex chat store](https://docs.llamaindex.ai/en/stable/module_guides/storing/chat_stores/))
+- when instrumenting GenAI client libraries that maintain conversation on the backend side (see [AWS Bedrock agent sessions](https://docs.aws.amazon.com/bedrock/latest/userguide/agents-session-state.html), [OpenAI Assistant threads](https://platform.openai.com/docs/api-reference/threads))
+Application developers that manage conversation history MAY add conversation id to GenAI and other spans or logs using custom span or log record processors or hooks provided by instrumentation libraries.
```

Comment:
nit this is rendering in the previous bullet point. It looks like the newline is getting lost, possibly because of the yaml folding `>`?

Reply:
Done